src/ActionList/Item.tsx

```diff
@@ -221,7 +221,9 @@ export const Item = React.forwardRef<HTMLLIElement, ActionListItemProps>(
                     id={labelId}
                     sx={{
                       flexGrow: slots.description && slots.description.props.variant !== 'block' ? 0 : 1,
-                      fontWeight: slots.description && slots.description.props.variant !== 'block' ? 'bold' : 'normal',
+                      fontWeight: slots.description ? 'bold' : 'normal',
+                      marginBlockEnd:
+                        slots.description && slots.description.props.variant !== 'inline' ? '4px' : undefined,
```

Comment:
should this be the token space-`1` than 4px?

Reply:
good catch!

Reply:
Can someone let me know what the syntax is meant to look like to access the spacing token here? I couldn't figure it out, gave up and used px 😛

Reply:
```diff
- slots.description && slots.description.props.variant !== 'inline' ? '4px' : undefined,
+ slots.description && slots.description.props.variant !== 'inline' ? 1 : undefined,
```

unlessssssss, styled-system doesn't understand logical properties like marginBlockEnd, only marginRight because the last release was in 2019 ([open issue](https://github.com/styled-system/styled-system/issues/1268))

Reply:
SIGH haha okay, thanks @siddharthkp!